requirements.txt

```diff
@@ -15,6 +15,7 @@ requests
 rich
 # scikit-fuzzy  # uncomment once fuzzy_logic/fuzzy_operations.py is fixed
 scikit-learn
+spacy
```

Comment:
The script relies on the en_core_web_md model from SpaCy, which is not explicitly mentioned in the requirements.txt.

Reply:
Fix this Issue. Thanks for Raising!